projects/plugins/jetpack/json-endpoints/jetpack/class.jetpack-json-api-plugins-endpoint.php

```diff
@@ -180,8 +181,9 @@ protected function format_plugin_v1_2( $plugin_file, $plugin_data ) {
 			$plugin['action_links'] = $action_link;
 		}
 
-		$autoupdate = $this->plugin_has_autoupdates_enabled( $plugin_file );
```

Comment:
If we don't use that method anymore, can we get rid of it entirely?

Reply:
Yes I don't see it being used anywhere else 👍   https://github.com/Automattic/jetpack/pull/18707/commits/4fc343f422bdd4967cefc2dbea44eab822062789